config/tipoff.php

```diff
@@ -30,12 +30,16 @@
 
         'company' => \Tipoff\Seo\Models\Company::class,
 
+        'company_user' => \Tipoff\Seo\Models\CompanyUser::class,
```

Comment:
We don't need to reference this as a model since it is really just a pivot table.